ssl/ssl_lib.c

```diff
@@ -2520,8 +2528,9 @@ int SSL_CTX_set_cipher_list(SSL_CTX *ctx, const char *str)
 {
     STACK_OF(SSL_CIPHER) *sk;
 
-    sk = ssl_create_cipher_list(ctx->method, &ctx->cipher_list,
-                                &ctx->cipher_list_by_id, str, ctx->cert);
+    sk = ssl_create_cipher_list(ctx->method, ctx->tls13_ciphersuites,
+                                &ctx->cipher_list, &ctx->cipher_list_by_id, str,
+                                ctx->cert);
```

Comment:
Can you explain what's happening here?  Is this parsing a single string for both sets of ciphers?

Reply:
Never mind, I think I see that`tls13_ciphersuites` is just passed through unaltered by the value of "str" and prepended to the cipher stack as-is.  What does this about the order of calls that configure TLS13 vs. TLS12 ciphers? Does this mean that the TLS13 configuration (if performed) has to happen first?